esp_encrypted_img/README.md

```diff
@@ -0,0 +1,10 @@
+# Encrypted Image Decryptor component
+
+This component is used for decryption of encrypted images used during OTA updates.
```

Comment:
```suggestion
This component provides an API interface to decrypt data defined in "ESP Encrypted Image" format. This format is as specified below:
```

This component can help in integrating pre encrypted firmware in over-the-air updates. Additionally, this component can also be used for other use-cases which requires addition of encryption layer for custom data.

Please add relevant image format diagram here.